dotnet/src/Connectors/Connectors.MistralAI/Client/ContentChunk.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.MistralAI.Client;
+
+[JsonDerivedType(typeof(TextChunk))]
+[JsonDerivedType(typeof(ImageUrlChunk))]
+internal abstract class ContentChunk(ContentChunkType type)
```

Comment:
Not sure why we need specific types for the chunks, cant we just use the Content chunk with both properties for now (that would simplify the code base, not sure if we need this specialization just for one field at each type.

```suggestion
internal abstract class ContentChunk(ContentChunkType type)
```